extensions/eclipse-che-theia-plugin-ext/src/node/che-plugin-service.ts

```diff
@@ -0,0 +1,120 @@
+/*********************************************************************
+ * Copyright (c) 2018 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+import { ChePluginService, ChePluginMetadata } from '../common/che-protocol';
+import { injectable, interfaces } from 'inversify';
+import axios, { AxiosInstance } from 'axios';
+
+// const { yaml } = require('js-yaml');
+const yaml = require('js-yaml');
+
+export interface ChePluginMetadataInternal {
```

Comment:
Maybe ChePluginMetadataInternal extends ChePluginMetadata?

Reply:
There is another `ChePluginMetadata` defined in `common/che-protocol`